### ISSUE TITLE
Fix more A11y violations

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#26350]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

This is to fix more a11y violations reported by cypress.Cypress

### Version

22.0.5

### Expected behavior

A11y violations are fixed.

### Actual behavior

A11y violations are detected and need to be fixed.

### How to Reproduce?

Various a11y violations as reported by cypress.Cypress
